docs/en/basechain-release-notes.md

```diff
@@ -8,6 +8,54 @@ sidebar_label: Basechain
 
 Reverse chronology order of release notes.
 
+## Pre-Basechain Build 1346 Hard Fork - 2019/11/26
+
+**Breaking changes:**
+- The `eth_getTransactionCount` and eth_call Web3 JSON-RPC methods on the `/eth` endpoint now expect
+  the input address to be an Ethereum account, not a Loom account.
+- `eth_getLogs` and any other Web3 JSON-RPC methods that query a range of blocks will now only allow
+  querying up to 20 blocks per request on public Extdev nodes.
+
+**New features:**
+- A node will now panic when it’s started using a jump-start that requires a newer build than the
```

Comment:
```suggestion
- A node will now panic when it’s started using a jump-start that requires a newer build than the node is currently running. Once you upgrade the binary, you can simply restart the node without restoring the working directory from the jump-start. This requires a jump-start that’s created after the `chaincfg:v1.4` feature flag is enabled, which means it will only work with future jump-starts (the first of which will be available a few days after this release).
```

Reply:
Can't see what changed, ignoring